Bulldozer.BinaryFile/Maps/MinistryDocument.cs

```diff
@@ -112,6 +112,9 @@ public int Map( ZipArchive folder, BinaryFileType ministryFileType )
                         documentForeignId = Regex.Match( filename, "\\d+$" ).Value;
                     }
 
+                    // Strip PMM from file name so it gets treated like all other background checks
+                    attributeName = attributeName.Equals( "PMMBackgroundCheck", StringComparison.OrdinalIgnoreCase ) ? "BackgroundCheck" : attributeName;
```

Comment:
```suggestion
                    attributeName = attributeName.ToLower().Contains( "backgroundcheck" ) ? "BackgroundCheck" : attributeName;
```

Reply:
@guymaness2 were there other changes necessary for this backgroundcheck attribute check or just this ToLower().Contains was it? I don't see any other notes, just wondering why we didn't take this.